ui_framework/src/components/gallery/gallery_item/gallery_item.js

```diff
@@ -2,18 +2,41 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-export const KuiGalleryItem = ({ children, className, ...rest }) => {
+const checkHrefAndOnClick = (props, propName, componentName) => {
```

Comment:
Nice. I should add this to K7 stuff too. I wasn't this detailed to check for both.